src/apps/lwaftr/fragmentv4_test.lua

```diff
@@ -13,19 +16,32 @@ local ffi = require("ffi")
 -- Returns a new packet, which contains an Ethernet frame, with an IPv4 header,
 -- followed by a payload of "payload_size" random bytes.
 --
-local function make_ipv4_packet(payload_size)
+local function make_ipv4_packet(payload_size, vlan_tag)
+   local eth_size = eth_proto:sizeof()
+   if vlan_tag and vlan_tag > 0 then
```

Comment:
though VID==0 is reserved, it is a valid VID value; please remove vlan_tag > 0 check